src/components/offers/list/OffersView.js

```diff
@@ -11,11 +11,14 @@ import {SorterWrapper} from "../sorter/SorterWrapper";
 import {ColumnAndDetailsLayout} from "../../commons/layouts/ColumnAndDetailsLayout";
 import {StandardViewAndFilterLayout} from "../../commons/layouts/StandardViewAndFilterLayout";
 import CenteredCircularProgress from "../../commons/CenteredCircularProgress";
+import Swal from "sweetalert2";
 
 export const OffersView = (props) => {
 
     const [selectedOffer, setSelectedOffer] = useState(null)
     const [offers, setOffers] = useState([])
+    const [fetchError, setFetchError] = useState(false)
+    const [fetching, setFetching] = useState(true)
```

Comment:
Fetching to powinno chyba być isFetching? Oraz takie pytanie czy nie zrobić funkcji okalającej w której byś ustawiał naraz odpowiednie wartości dla fetchError i fetching?